app/controllers/journalist/articles_controller.rb

```diff
@@ -0,0 +1,46 @@
+class Journalist::ArticlesController < ApplicationController
+    before_action :check_journalist
+    before_action :authenticate_user!
+    def index
+        @articles = Article.all.where(user_id: current_user.id)
+    end
+
+    def show
+        @article = Article.find(params[:id])
+    end
+    
+    def edit
```

Comment:
Set a conditional that the user(journalist) can only access this article if he is the owner.

Reply:
done